dashboard/app/views/devise/sessions/new.html.haml

```diff
@@ -44,7 +45,7 @@
           %button#signup-button.secondary{type: 'button'}= t('nav.user.signup')
 
   .span4
-    .sidebar= render "devise/shared/links"
+    .sidebar= render "devise/shared/links" unless @hide_oauth_sidebar
```

Comment:
Yes, my goal was students don't try to go through that flow again.  I am fine with adding it back in with the caveat that if we still see a lot of these errors, we should consider removing it then.  Are you both okay with that?